modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+
+=== The formatting of contenteditable="false" elements are no longer cloned to new cells while creating new table rows
+// TINY-9449
+
+In previous versions of {productname}, contenteditable="false" elements such as a *merge tags* could clone the styling to a new table row. In some situations, this cloning would happen during the creation of new table row, where the merge tag prefix would unexpectedly clone the formatting from the previous cell to the new cell.
+
+To correct this issue, {productname} added changes to the *clone logic*, and the cursor position API, which will now return `true`, for any passed `contenteditable=false` elements.
+
+In {productname} 6.4, the formatting of `contenteditable="false"` elements are no longer cloned to new cells while creating new table rows.
+
+=== Changed the color of @dialog-table-border-color, and added right padding to the first cell of dialog table
+// TINY-9380
+
+An area for improvement was identified during an accessibility audit for the *help menu* within {productname}, which focuses on users that rely on **zoom orient** to help with visibility.
+
+The consequence of the previous UX, meant that adjustments needed to be made to the horizontal lines within the *help menu* table, that seperate each `row`.
+
+To fix this accessibility issue, {productname} 6.4, increased the contrast ratio to `3:1` to each line divider, and changed the `line` color to `#646D78` to further help define were the separations occur in the table.
+
+As a result, the `help menu` is now styled to support a wider range of users, by these new changes implemented.
+
+=== The sidebar element now has accessibility role region when visible and presentation when hidden
+// TINY-9517
+
+In previous versions of {productname}, Axe-DevTools (web accessibility testing extension) identified that the sidebar element within {productname} was set to `role="complementary"` when the element was either in a visible or hidden state, which created issues with screen readers, as the general rule is that landmarks should not be contained within another landmark.
+
+As a consequence by having the sidebar set to `role="complementary"` for both states, tools such as screen readers in some situations would disable their functionality.
+
+This is important as when a sidebar element is visible, it has an accessibility role of `region`. This ensures the sidebar and its contents are accessible to screen readers and other accessibility tools. This role is important for screen readers to identify the sidebar as a meaningful part of the page. When the sidebar element is hidden, it has a role of presentation, which is used to inform the user that the element is not important and can be safely ignored. 
+
+To fix this issue, changes were made to the role attribute of the sidebar element `(.tox-sidebar)`, which is now set to `role="region"` when it's active state is visible, otherwise it defaults to `role="presentation"` when hidden.
```

Comment:
```suggestion
To fix this issue, changes were made to the role attribute of the sidebar element, which is now set to `role="region"` when it's active state is visible, otherwise it defaults to `role="presentation"` when hidden.
```